src/openrct2/rct1/S4Importer.cpp

```diff
@@ -98,6 +98,7 @@ class S4Importer final : public IParkImporter
     const utf8 * _s4Path = nullptr;
     rct1_s4      _s4 = { 0 };
     uint8        _gameVersion = 0;
+    uint8        _parkValueConversionFactor = 0;
```

Comment:
Should the conversion factor not be recalculated each time a park is loaded? Storing it here could mean the factor is reused for another park.